src/Bab/RabbitMq/VhostManager.php

```diff
@@ -180,14 +180,19 @@ private function createQueues(Configuration $config)
             }
 
             for ($i = 0; $i < count($retries); $i++) {
+                if ($i === 0) {
```

Comment:
Ok so to sum up, we create exhanges like retry and delay in the `createBaseStructure` based on the fact that a `delay` or `retries` parameter is specified in at least one queue configuration. And remove all exchange creation in `createQueues`. 
I'm okay with that :)